config/schema/oe_list_pages.schema.yml

```diff
@@ -21,6 +21,12 @@ node.type.*.third_party.oe_list_pages:
       label: 'The view mode to use when rendering the node in lists'
     default_sort:
       type: oe_list_pages.list_sort
+    default_exposed_filters:
```

Comment:
Media don't need default exposed filters?

Also, seeing that the entire third party setting is the same across the two, we can extract the schema to its own thing no? Like I did with the sort.